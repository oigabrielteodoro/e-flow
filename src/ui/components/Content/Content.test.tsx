import React from 'react'
import { render, screen } from '@testing-library/react'

import { Content } from '.'

describe('<Content />', () => {
  it('should render correctly', () => {
    const { container } = render(<Content />)

    expect(screen.getByText('Dashboard')).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
