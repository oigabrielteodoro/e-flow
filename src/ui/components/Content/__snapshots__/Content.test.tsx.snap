// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> should render correctly 1`] = `
.c6 {
  position: relative;
}

.c6:hover span {
  opacity: 1;
  visibility: visible;
}

.c6 > span {
  color: #ffffff;
  background: #0047BB;
  border-radius: 0.4rem;
  padding: 0.5rem 1.2rem;
  white-space: nowrap;
  -webkit-transition: all 0.5s;
  transition: all 0.5s;
  opacity: 0;
  visibility: hidden;
  position: absolute;
  left: 50%;
  -webkit-transform: translateX(-50%);
  -ms-transform: translateX(-50%);
  transform: translateX(-50%);
  top: calc(-100% + -1.7rem);
  z-index: 1;
}

.c6 > span::before {
  content: '';
  position: absolute;
  border-style: solid;
  border-width: 0.9rem 0.9rem 0 0.9rem;
  border-color: #0047BB transparent transparent transparent;
  left: 50%;
  -webkit-transform: translateX(-50%);
  -ms-transform: translateX(-50%);
  transform: translateX(-50%);
  bottom: -0.9rem;
}

.c0 {
  background: #f5f8fa;
  border-radius: 2.4rem 0 0 0;
  padding: 3.2rem 2rem;
}

.c0 header {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 header img {
  width: 2.4rem;
  height: 2.8rem;
}

.c0 header h1 {
  margin-left: 1.1rem;
  font-size: 2.4rem;
  font-weight: 600;
}

.c1 {
  margin-top: 3.2rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: #ffffff;
  margin-right: auto;
}

.c2 input {
  border: 0;
  height: 4rem;
  background: transparent;
  border-radius: 0.8rem 0 0 0.8rem;
  border: 0.1rem solid #e1e0e7;
  border-right-color: transparent;
  padding: 0 1.6rem;
}

.c2 input::-webkit-input-placeholder {
  color: #acacac;
}

.c2 input::-moz-placeholder {
  color: #acacac;
}

.c2 input:-ms-input-placeholder {
  color: #acacac;
}

.c2 input::placeholder {
  color: #acacac;
}

.c2 button {
  border: 0;
  width: 4rem;
  height: 4rem;
  border-radius: 0.8rem;
  background: #0047BB;
  color: #ffffff;
  margin-left: -0.8rem;
  -webkit-transition: filter 0.2s;
  transition: filter 0.2s;
}

.c2 button:hover {
  -webkit-filter: brightness(0.85);
  filter: brightness(0.85);
}

.c3 {
  max-width: 74.8rem;
  margin-top: 2.4rem;
  padding: 3rem 2.3rem;
  border-radius: 0.8rem;
  background: #ffffff;
  box-shadow: 0 0.5rem 1.2rem rgba(222,222,231,0.4);
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c5 fieldset {
  border: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: 1.2rem;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

.c5 fieldset strong {
  font-weight: 500;
}

.c5 fieldset span {
  color: #acacac;
  font-size: 1.4rem;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
}

.c7 img {
  margin-right: 0.8rem;
}

.c7 span {
  margin-top: 0.8rem;
  color: #d64b45;
  font-weight: 500;
}

<div
  class="c0"
>
  <header>
    <img
      alt="Icon Dashboard"
      src="test-file-stub"
    />
    <h1>
      Dashboard
    </h1>
  </header>
  <main
    class="c1"
  >
    <div
      class="c2"
    >
      <input
        placeholder="Buscar empresa"
      />
      <button>
        <img
          alt="Icon Search"
          src="test-file-stub"
        />
      </button>
    </div>
    <section
      class="c3"
    >
      <div
        class="c4"
      >
        <div
          class="c5"
        >
          <div
            class="c6"
          >
            <span>
              Adicionar aos favoritos
            </span>
            <img
              alt="Icon Star"
              src="test-file-stub"
            />
          </div>
          <fieldset>
            <strong>
              MSFT
            </strong>
            <span>
              Microsoft
            </span>
          </fieldset>
        </div>
        <div
          class="c7"
        >
          <div>
            <img
              alt="Icon Down Pricing"
              src="test-file-stub"
            />
            <strong>
              $265,42
            </strong>
          </div>
          <span>
            $-0.09 (-0.03%)
          </span>
        </div>
      </div>
      <div
        class="recharts-wrapper"
        style="position: relative; cursor: default; width: 702px; height: 274px;"
      >
        <svg
          class="recharts-surface"
          height="274"
          version="1.1"
          viewBox="0 0 702 274"
          width="702"
        >
          <defs>
            <clippath
              id="recharts1-clip"
            >
              <rect
                height="214"
                width="642"
                x="60"
                y="30"
              />
            </clippath>
          </defs>
          <defs>
            <lineargradient
              id="gradient"
              x1="100%"
              x2="0%"
              y1="0%"
              y2="100%"
            >
              <stop
                offset="40%"
                stop-color="#0047BB"
                stop-opacity="0.1"
              />
              <stop
                offset="80%"
                stop-color="#ffffff"
                stop-opacity="0.1"
              />
            </lineargradient>
          </defs>
          <g
            class="recharts-cartesian-grid"
          >
            <g
              class="recharts-cartesian-grid-horizontal"
            >
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="60"
                x2="702"
                y="30"
                y1="244"
                y2="244"
              />
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="60"
                x2="702"
                y="30"
                y1="190.5"
                y2="190.5"
              />
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="60"
                x2="702"
                y="30"
                y1="137"
                y2="137"
              />
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="60"
                x2="702"
                y="30"
                y1="83.5"
                y2="83.5"
              />
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="60"
                x2="702"
                y="30"
                y1="30"
                y2="30"
              />
            </g>
            <g
              class="recharts-cartesian-grid-vertical"
            >
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="60"
                x2="60"
                y="30"
                y1="30"
                y2="244"
              />
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="131.33333333333331"
                x2="131.33333333333331"
                y="30"
                y1="30"
                y2="244"
              />
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="202.66666666666666"
                x2="202.66666666666666"
                y="30"
                y1="30"
                y2="244"
              />
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="274"
                x2="274"
                y="30"
                y1="30"
                y2="244"
              />
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="345.3333333333333"
                x2="345.3333333333333"
                y="30"
                y1="30"
                y2="244"
              />
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="416.66666666666663"
                x2="416.66666666666663"
                y="30"
                y1="30"
                y2="244"
              />
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="488"
                x2="488"
                y="30"
                y1="30"
                y2="244"
              />
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="559.3333333333333"
                x2="559.3333333333333"
                y="30"
                y1="30"
                y2="244"
              />
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="630.6666666666666"
                x2="630.6666666666666"
                y="30"
                y1="30"
                y2="244"
              />
              <line
                fill="none"
                height="214"
                offset="[object Object]"
                stroke="#f5f8fa"
                width="642"
                x="60"
                x1="702"
                x2="702"
                y="30"
                y1="30"
                y2="244"
              />
            </g>
          </g>
          <g
            class="recharts-layer recharts-cartesian-axis recharts-yAxis yAxis"
          >
            <g
              class="recharts-cartesian-axis-ticks"
            >
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="214"
                  orientation="left"
                  stroke="none"
                  text-anchor="end"
                  type="number"
                  width="60"
                  x="48"
                  y="244"
                >
                  <tspan
                    dy="0.355em"
                    x="48"
                  >
                    $0
                  </tspan>
                </text>
              </g>
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="214"
                  orientation="left"
                  stroke="none"
                  text-anchor="end"
                  type="number"
                  width="60"
                  x="48"
                  y="190.5"
                >
                  <tspan
                    dy="0.355em"
                    x="48"
                  >
                    $250
                  </tspan>
                </text>
              </g>
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="214"
                  orientation="left"
                  stroke="none"
                  text-anchor="end"
                  type="number"
                  width="60"
                  x="48"
                  y="137"
                >
                  <tspan
                    dy="0.355em"
                    x="48"
                  >
                    $500
                  </tspan>
                </text>
              </g>
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="214"
                  orientation="left"
                  stroke="none"
                  text-anchor="end"
                  type="number"
                  width="60"
                  x="48"
                  y="83.5"
                >
                  <tspan
                    dy="0.355em"
                    x="48"
                  >
                    $750
                  </tspan>
                </text>
              </g>
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="214"
                  orientation="left"
                  stroke="none"
                  text-anchor="end"
                  type="number"
                  width="60"
                  x="48"
                  y="30"
                >
                  <tspan
                    dy="0.355em"
                    x="48"
                  >
                    $1000
                  </tspan>
                </text>
              </g>
            </g>
          </g>
          <g
            class="recharts-layer recharts-cartesian-axis recharts-xAxis xAxis"
          >
            <g
              class="recharts-cartesian-axis-ticks"
            >
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="30"
                  orientation="bottom"
                  stroke="none"
                  text-anchor="left"
                  type="category"
                  width="642"
                  x="60"
                  y="260"
                >
                  <tspan
                    dy="0.71em"
                    x="60"
                  >
                    09:00
                  </tspan>
                </text>
              </g>
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="30"
                  orientation="bottom"
                  stroke="none"
                  text-anchor="left"
                  type="category"
                  width="642"
                  x="131.33333333333331"
                  y="260"
                >
                  <tspan
                    dy="0.71em"
                    x="131.33333333333331"
                  >
                    10:00
                  </tspan>
                </text>
              </g>
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="30"
                  orientation="bottom"
                  stroke="none"
                  text-anchor="left"
                  type="category"
                  width="642"
                  x="202.66666666666666"
                  y="260"
                >
                  <tspan
                    dy="0.71em"
                    x="202.66666666666666"
                  >
                    10:30
                  </tspan>
                </text>
              </g>
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="30"
                  orientation="bottom"
                  stroke="none"
                  text-anchor="left"
                  type="category"
                  width="642"
                  x="274"
                  y="260"
                >
                  <tspan
                    dy="0.71em"
                    x="274"
                  >
                    11:00
                  </tspan>
                </text>
              </g>
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="30"
                  orientation="bottom"
                  stroke="none"
                  text-anchor="left"
                  type="category"
                  width="642"
                  x="345.3333333333333"
                  y="260"
                >
                  <tspan
                    dy="0.71em"
                    x="345.3333333333333"
                  >
                    12:00
                  </tspan>
                </text>
              </g>
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="30"
                  orientation="bottom"
                  stroke="none"
                  text-anchor="left"
                  type="category"
                  width="642"
                  x="416.66666666666663"
                  y="260"
                >
                  <tspan
                    dy="0.71em"
                    x="416.66666666666663"
                  >
                    13:00
                  </tspan>
                </text>
              </g>
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="30"
                  orientation="bottom"
                  stroke="none"
                  text-anchor="left"
                  type="category"
                  width="642"
                  x="488"
                  y="260"
                >
                  <tspan
                    dy="0.71em"
                    x="488"
                  >
                    14:00
                  </tspan>
                </text>
              </g>
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="30"
                  orientation="bottom"
                  stroke="none"
                  text-anchor="left"
                  type="category"
                  width="642"
                  x="559.3333333333333"
                  y="260"
                >
                  <tspan
                    dy="0.71em"
                    x="559.3333333333333"
                  >
                    15:00
                  </tspan>
                </text>
              </g>
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="30"
                  orientation="bottom"
                  stroke="none"
                  text-anchor="left"
                  type="category"
                  width="642"
                  x="630.6666666666666"
                  y="260"
                >
                  <tspan
                    dy="0.71em"
                    x="630.6666666666666"
                  >
                    16:30
                  </tspan>
                </text>
              </g>
              <g
                class="recharts-layer recharts-cartesian-axis-tick"
              >
                <text
                  class="recharts-text recharts-cartesian-axis-tick-value"
                  color="#657786"
                  fill="#666"
                  font-size="11"
                  height="30"
                  orientation="bottom"
                  stroke="none"
                  text-anchor="left"
                  type="category"
                  width="642"
                  x="702"
                  y="260"
                >
                  <tspan
                    dy="0.71em"
                    x="702"
                  >
                    17:00
                  </tspan>
                </text>
              </g>
            </g>
          </g>
          <g
            class="recharts-layer recharts-line"
          >
            <path
              class="recharts-curve recharts-line-curve"
              d="M60,244C83.77777777777777,195.85000000000002,107.55555555555554,147.70000000000002,131.33333333333331,115.60000000000001C155.1111111111111,83.5,178.88888888888889,51.39999999999999,202.66666666666666,51.39999999999999C226.44444444444443,51.39999999999999,250.22222222222223,72.79999999999998,274,72.79999999999998C297.77777777777777,72.79999999999998,321.55555555555554,51.39999999999999,345.3333333333333,51.39999999999999C369.1111111111111,51.39999999999999,392.88888888888886,244,416.66666666666663,244C440.4444444444444,244,464.22222222222223,199.06000000000003,488,199.06000000000003C511.77777777777777,199.06000000000003,535.5555555555555,211.9,559.3333333333333,211.9C583.1111111111111,211.9,606.8888888888888,179.79999999999998,630.6666666666666,179.79999999999998C654.4444444444445,179.79999999999998,678.2222222222222,188.35999999999999,702,196.92"
              fill="none"
              height="214"
              points="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]"
              stroke="#0047BB"
              stroke-dasharray="0px 0px"
              stroke-width="1"
              type="monotone"
              width="642"
            />
          </g>
          <g
            class="recharts-layer recharts-area"
          >
            <g
              class="recharts-layer"
            >
              <defs>
                <clippath
                  id="animationClipPath-recharts-area-5"
                >
                  <rect
                    height="246"
                    width="0"
                    x="60"
                    y="0"
                  />
                </clippath>
              </defs>
              <g
                class="recharts-layer"
                clip-path="url(#animationClipPath-recharts-area-5)"
              >
                <g
                  class="recharts-layer"
                >
                  <path
                    class="recharts-curve recharts-area-area"
                    d="M60,244C83.77777777777777,195.85000000000002,107.55555555555554,147.70000000000002,131.33333333333331,115.60000000000001C155.1111111111111,83.5,178.88888888888889,51.39999999999999,202.66666666666666,51.39999999999999C226.44444444444443,51.39999999999999,250.22222222222223,72.79999999999998,274,72.79999999999998C297.77777777777777,72.79999999999998,321.55555555555554,51.39999999999999,345.3333333333333,51.39999999999999C369.1111111111111,51.39999999999999,392.88888888888886,244,416.66666666666663,244C440.4444444444444,244,464.22222222222223,199.06000000000003,488,199.06000000000003C511.77777777777777,199.06000000000003,535.5555555555555,211.9,559.3333333333333,211.9C583.1111111111111,211.9,606.8888888888888,179.79999999999998,630.6666666666666,179.79999999999998C654.4444444444445,179.79999999999998,678.2222222222222,188.35999999999999,702,196.92L702,244C678.2222222222222,244,654.4444444444445,244,630.6666666666666,244C606.8888888888888,244,583.1111111111111,244,559.3333333333333,244C535.5555555555555,244,511.77777777777777,244,488,244C464.22222222222223,244,440.4444444444444,244,416.66666666666663,244C392.88888888888886,244,369.1111111111111,244,345.3333333333333,244C321.55555555555554,244,297.77777777777777,244,274,244C250.22222222222223,244,226.44444444444443,244,202.66666666666666,244C178.88888888888889,244,155.1111111111111,244,131.33333333333331,244C107.55555555555554,244,83.77777777777777,244,60,244Z"
                    fill="url(#gradient)"
                    fill-opacity="1"
                    height="214"
                    points="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]"
                    stroke="none"
                    stroke-width="2"
                    type="monotone"
                    width="642"
                  />
                  <path
                    class="recharts-curve recharts-area-curve"
                    d="M60,244C83.77777777777777,195.85000000000002,107.55555555555554,147.70000000000002,131.33333333333331,115.60000000000001C155.1111111111111,83.5,178.88888888888889,51.39999999999999,202.66666666666666,51.39999999999999C226.44444444444443,51.39999999999999,250.22222222222223,72.79999999999998,274,72.79999999999998C297.77777777777777,72.79999999999998,321.55555555555554,51.39999999999999,345.3333333333333,51.39999999999999C369.1111111111111,51.39999999999999,392.88888888888886,244,416.66666666666663,244C440.4444444444444,244,464.22222222222223,199.06000000000003,488,199.06000000000003C511.77777777777777,199.06000000000003,535.5555555555555,211.9,559.3333333333333,211.9C583.1111111111111,211.9,606.8888888888888,179.79999999999998,630.6666666666666,179.79999999999998C654.4444444444445,179.79999999999998,678.2222222222222,188.35999999999999,702,196.92"
                    fill="none"
                    fill-opacity="1"
                    height="214"
                    points="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]"
                    stroke="#3182bd"
                    stroke-width="2"
                    type="monotone"
                    width="642"
                  />
                </g>
              </g>
            </g>
          </g>
        </svg>
        <div
          class="recharts-tooltip-wrapper"
          style="transform: translate(undefinedpx, undefinedpx); pointer-events: none; visibility: hidden; position: absolute; top: 0px; left: 0px;"
        />
      </div>
    </section>
  </main>
</div>
`;
